cmd/root.go

```diff
@@ -535,10 +706,18 @@ func updateComment(ghComment github.IssueComment, jComment jira.Comment, jIssue
 		return nil
 	}
 
-	user, _, err := ghClient.Users.Get(context.Background(), *ghComment.User.Login)
+	u, _, err := makeGHRequest(func() (interface{}, *github.Response, error) {
+		return ghClient.Users.Get(context.Background(), *ghComment.User.Login)
+	})
 	if err != nil {
```

Comment:
This is really weird, there should not be a period between these two clauses but rather a colon. Conventionally we do:
```
log.Errorf("error retrieving GitHub user %s: %v", user.Login, err)
```